exes/mqtt-broadcaster/CMakeLists.txt

```diff
@@ -25,29 +30,29 @@ target_link_libraries(mqtt-broadcaster
     tfc::sparkplug::proto
 )
 
-target_compile_definitions(mqtt-broadcaster
+target_compile_definitions(mqtt_broadcaster
   PUBLIC
     ASYNC_MQTT_USE_TLS
 )
 
-target_compile_options(mqtt-broadcaster PRIVATE -Wno-exit-time-destructors)
+target_compile_options(mqtt_broadcaster PRIVATE -Wno-exit-time-destructors)
```

Comment:
Where is this needed, it should only be applied to that very line.

Reply:
It's not needed, this is something old, taking the line out.